src/database/seeders/20230421014222-wishlist.js

```diff
@@ -0,0 +1,80 @@
+import { v4 as uuidv4 } from 'uuid';
+import { hashPassword } from '../../utils/password';
+
+const userId = '5c2918e4-9482-412a-9c30-1acd08cb5dbb';
+// eslint-disable-next-line valid-jsdoc
+/** @type {import('sequelize-cli').Migration} */
+export async function up(queryInterface) {
+  const collectionId = await queryInterface.rawSelect(
+    'collections',
+    {
+      where: {
+        name: 'testing Collection',
+      },
+    },
+    ['id']
+  );
+  const [resultOne] = await queryInterface.sequelize.query(`
+  SELECT COUNT(*) AS count FROM users WHERE email='testingwishlist@example.com'
+`);
+  // eslint-disable-next-line eqeqeq
+  if (resultOne[0].count == 0) {
+    await queryInterface.bulkInsert('users', [
+      {
```

Comment:
I think here you can create variable above for this password and then you call them below,

Reply:
requested change done 👍